# Humans of BioSciences post


## Who are you and what do you do in the School?

Iâ€™m Jeff Paril, a research associate in weed genomics in the Adaptive Evolution Lab working with Alex Fournier-Level. Also, I've recently concluded a Post-doc research position in the Spatial Ecology and Evolution Lab with Ben Phillips where we chased suppression gene drives in space using computer simulations.

This is me in a wheat field somewhere in rural Victoria during a ryegrass seed collection trip:
![](/img/IMG20181111092142.jpg)
![](/img/IMG20181108114219.jpg)

## What problems are you trying to solve with your research?

I try to dabble in the basic question in genetics, which is how do we predict the phenotype of an organisms, i.e. the physical, morphological, biochemical or any observable characteristics, using its genotype, i.e. the sequence of bases A, T, C, and G in its DNA. I use statistical techniques to predict the levels of herbicide resistance of ryegrass. Ryegrass is a weed which is considered as a major agricultural pest in wheat and barley cropping. Farmers hate these weeds because these can significantly reduce their crop yields. They use herbicides which are specially formulated chemical concoctions to kill these weeds with nil to minimal non-target effects on the environment. However, these weeds are very crafty! They're evolving right in front of our eyes, by developing herbicide resistance over decades of herbicide application!

And so as the saying goes,"if you know the enemy and know yourself, you need not fear the result of a hundred battles" (Sun Tzu, The Art of War), to effectively and efficiently fight the negative effects of weeds on crop yields we need a method of measuring the levels of herbicide resistance of weed populations. Our weed genomics project aim to bring weed herbicide resistance testing into the 21st century using genomics and genomic prediction. We are gathering phenotype data by performing herbicide resistance assays on more than a hundred ryegrass populations collected across South-East Australia. We are then extracting the DNA of these populations for genotyping. Finally, we train or build our genomic prediction models using these phenotype and genotype data. With these models we aim predict with high accuracy the levels of herbicide resistance of new weed populations. This has the potential to improve not only the accuracy of the assays but also the speed and throughput. For example, if a farmer wants to test the ryegrass population in one of his paddocks to 10 different herbicides, instead of performing 10 different herbicide assays, we plan on simply genotyping the population once, then quickly and accurately predict the resistances.

By providing timely information on the levels of herbicide resistance of weed populations in the field, we can help farmers and agronomists make informed decisions on the best way to control weeds. For example, if we predict high resistance for an herbicide, let's say the infamous glyphosate or roundup herbicide, then the farmers may opt to reduce if not totally avoid using glyphosate in favour of another herbicide the weeds have not developed resistance to yet. Alternatively, the farmer may develop a hybrid approach by using both herbicides and non-chemical weed control strategies such as hot water/steam treatments.

Herbicide application is a chemical method of weed population control. Other methods exists such as mechanical methods where tools are used such as ploughs and rotavators, and physical methods including the use of high temperature in the form of steam or fire. A new promising strategy is the genetic control method, where we use suppression suppression gene drives to eradicate populations.

Suppression gene drive is a new and exciting application of gene-editing technologies (e.g. CRISPR-Cas9). However, instead of treating human diseases, the objective is to eradicate (or control) pest and invasive populations such as the dengue- and malaria-carrying mosquitoes, and cane toads in Australia. We have confirmed that simplistic models which assume random mating is insufficient to predict the success or failure of these gene drives. We need to account for the movement of these noxious species across landscapes because depending on the species, population, and type of gene drive, the target population may never be eradicated. This means that the suppression gene drive will chase the target population and never succeed because the pest population is too fast or the target individuals can penetrate the wave of invading gene drive-carriers and escape eradication.

## What skills do you bring to bear on these problems?

I try to balance experimental and theoretical work. I perform greenhouse experiments, field samplings, wet lab experiments, and computational/in-silico experiments. I'm using computer vision to extract information from 2D and 3D hyperspectral plant representations, i.e. photographs and 3D scans. These included seed germination and seedling characteristics, as well chlorosis and necrosis measurements and general plant health metrics. I also take part in genomics, specifically sequencing, assembly and annotation of the *Lolium rigidum* genome. I've also contributed to the development of spatially explicit computer simulations for suppression gene drives, and the derivation of a recursive formula for suppression gene drive allele frequencies.

## Do you have any outlandish stories from your time here at Unimelb?

There were very smart mice in the old glasshouses in BioSciences 4 before the renovation. They only ate my ryegrass plants instead of the strawberries, and I thought that's very health-conscious of them. They'd rather have a high-fibre rather than a high-sugar diet. I hope they found a new home that's not in the new glasshouse complex where I'm growing my plants.

## Do you have any advice for current students?

Make mistakes constantly and learn from them constantly. Focus on activities you find fun and which keep you motivated. Learn new things and be versatile as much as you can.

## Links:
- I sometimes write my thoughts [here](https://jeffersonfparil.github.io).
- You may also find my tools useful:
  + [violinplotter](https://github.com/jeffersonfparil/violinplotter) to plot and compare means with violin plots.
  + [GWAlpha.jl](https://github.com/jeffersonfparil/GWAlpha.jl), a Julia package for building genomic prediction models and performing genome-wide association on quantitative traits by inferring additive allelic effects using pool sequencing data (Pool-seq; i.e. allele frequencies).
  + [mucilage_seed_areas_openCV](https://gitlab.com/jeffersonfparil/mucilage_seed_areas_opencv) for quantifying Arabidopsis seed mucilage area with OpenCV in python.
  + [kungAnikAnik](https://github.com/jeffersonfparil/kungAnikAnik), a collection of scalable vector graphics files under Creative Commons Attribution-NonCommercial 4.0 International License.